packages/components/Shimmer/FURNShimmer.podspec

```diff
@@ -0,0 +1,17 @@
+Pod::Spec.new do |s|
+  s.name             = 'FURNShimmer'
```

Comment:
Naming is going to be a lot of fun here. I just chose this to get us started

Reply:
I still don't like this name. I wonder if podspecs can have names like "FluentUI-ReactNative/Shimmer"?